.github/workflows/release.yml

```diff
@@ -286,7 +288,7 @@ jobs:
             org.opencontainers.image.licenses=${{ github.event.repository.license.spdx_id }}
 
   docker-standalone:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
This can totally be a followup PR, but I think it makes sense to have proper [two-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) for our docker images (moonbase-standalone, moonbase-parachain, polkadot).

It will have these advantages:
* It totally decouples the docker workflow from our unit test and integration test workflow, so they can be parallelized.
* It allows users to build the docker images themselves with `docker build`

Reply:
The docker generation is < 1 minute, I don't see much benefit of parallelization for those.